eclipse-collections/src/main/java/org/eclipse/collections/impl/set/AbstractUnifiedSet.java

```diff
@@ -430,4 +435,28 @@ public boolean isProperSubsetOf(SetIterable<? extends T> candidateSuperset)
         this.forEach(new NonMutatingAggregationProcedure<T, K2, V>(map, groupBy, zeroValueFactory, nonMutatingAggregator));
         return map;
     }
+
+    public <V> MutableObjectLongMap<V> sumByInt(Function<? super T, ? extends V> groupBy, IntFunction<? super T> function)
```

Comment:
At an initial glance, yes it should. I can investigate and make the change as a part of a different PR.